index.html

```diff
@@ -1,12 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="Portfolio website">
+  <meta name="keywords" content="HTML, CSS">
+  <meta name="author" content="Cedric KOSSI">
   <link rel="stylesheet" href="main.css">
   <title>Cedric KOSSI</title>
 </head>
+
 <body>
-<h1>Hello Microverse!</h1>
+  <header class="navbar">
+    <nav class="container">
+      <a class="logo" href="#">My Logo</a>
+      <img src="img/Union.png" alt="Navigation">
+    </nav>
+  </header>
+  <section class="hero">
+    <div class="container">
+      <h1 class="fsthead">I’m Cedric
+        Glad to see you!</h1>
+    </div>
+    <p class="intro">I’m a software developer! I can help you build a product , feature or website Look through some
+      of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+      me.</p>
+    <div class="cnt-txt">LET’S CONNECT</div>
+    <div class="ic-container">
+      <a href=""><img class="ic-social-media" src="img/twitter.png" alt="twitter"></a>
+      <a href=""><img class="ic-social-media" src="img/linkedin_icon.png" alt="linkedIn"></a>
+      <a href=""><img class="ic-social-media" src="img/medium.png" alt="medium"></a>
+      <a href=""><img class="ic-social-media" src="img/github.png" alt="github"></a>
+      <a href=""><img class="ic-social-media" src="img/vec.png" alt="vector"></a>
+    </div>
```

Comment:
- [ ] You did an amazing job wrapping icons inside of links but please as the previous reviewer said you have to put them inside of ul. Kindly change that. Keep rocking 💪